### DISCUSSION TITLE
how to install the snapshot

### DISCUSSION THREAD

**Discussion by brixxnatt** [ant-media/Ant-Media-Server#6240] [General]
hello , please how to install the snapshot , anyone can help in details how to install it

**Comment by muratugureminoglu** [comment]
Hi @brixxnatt 

Snapshot version is only available in enterprise version. If you are asking how to access this package, you can start a trial at https://antmedia.io and get this package. 

If you are asking about installation, just follow the link below. 

https://antmedia.io/docs/quick-start/

or 

```
wget -O install_ant-media-server.sh https://raw.githubusercontent.com/ant-media/Scripts/master/install_ant-media-server.sh && sudo chmod 755 install_ant-media-server.sh
sudo ./install_ant-media-server.sh -i <ANT_MEDIA_SERVER_ZIP_FILE>
```

Regards.
